src/utils/pgwire/src/pg_message.rs

```diff
@@ -43,11 +43,14 @@ pub struct FeQueryMessage {
 }
 
 impl FeQueryMessage {
-    pub fn get_sql(&self) -> &str {
+    pub fn get_sql(&self) -> Result<&str> {
         // Why there is a \0..
         match std::str::from_utf8(&self.sql_bytes[..]) {
-            Ok(v) => v.trim_end_matches('\0'),
-            Err(e) => panic!("Invalid UTF-8 sequence: {}", e),
+            Ok(v) => Ok(v.trim_end_matches('\0')),
```

Comment:
Maybe we'd be better off using `CStr` here? cc @BowenXiao1999 
```rust
CStr::from_bytes_with_nul(&self.sql_bytes)?.to_str()?
```

Reply:
+1